### PR TITLE
Fix some keyword duplicates due to hyphenation, casing, plural, and so on.

### PR DESCRIPTION
**Pull request by silene**

### CODE REVIEW

**Comment by herbelin** [comment]
Note that for the packages called "contribs", the keywords were generated (afaik) from the `description` files of the contribution (e.g. [here](https://github.com/coq-contribs/euclidean-geometry/blob/master/description)). I don't know the exact status of the generating tools though.

File: extra-dev/packages/coq-color/coq-color.8.7.dev/opam [hunk 1]
```diff
@@ -71,7 +70,7 @@ tags: [
   "keyword:string"
 
   "keyword:alpha-equivalence"
-  "keyword:de Bruijn indices"
+  "keyword:de bruijn indices"
```
Comment (gares) [inline extra-dev/packages/coq-color/coq-color.8.7.dev/opam]: Hum, this is a name... are you sure?

Comment (silene) [reply extra-dev/packages/coq-color/coq-color.8.7.dev/opam]: I went by the majority rule. Feel free to commit a huge patch to capitalize all the keywords involving names.

**Comment by silene** [comment]
> the keywords were generated (afaik) from the description files of the contribution (e.g. here).

I don't think the plan is to ever regenerate them from description files, so it should not matter.

**Comment by herbelin** [comment]
> I don't think the plan is to ever regenerate them from description files, so it should not matter.

I don't see how to do differently than considering the packages at coq-contribs to be the master source of the opam packages, since it is where all the source information is centralized, hence to keep the keywords synchronized there with the keywords here.

**Comment by gares** [comment]
> I don't think the plan is to ever regenerate them

I agree

> I don't see how to do differently than ...

I see a way: remove the metadata from the coq-contribs git repos and keep it
just in the opam file.

This has 2 advantages:
- no need to convert between formats, just copy the file to synchronize (if you want to)
- if you put the opam file in the root of your git sources, then
  `opam add pin mything https://github.com/my/thing.git`
is valid and installs the package even if the package is not part of an opam archive. This is standard practice.

**Comment by herbelin** [comment]
> if you put the opam file in the root of your git sources, then
`opam add pin mything https://github.com/my/thing.git`
is valid and installs the package even if the package is not part of an opam archive. This is standard practice.

Yes, I'm supporting this also. How to proceed then?

**Pull request merged**